CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- fail_on_diagnostics toml flag
```

Comment:
Best to include the jira ticket, as seen in the other lines.

Also you can monospace `fail_on_diagnostics` since this is markdown, and specify `snooty.toml`, since there's a number of toml files that snooty ingests.